doc/Developer Guide/Plugin Packaging and Versioning/Readme.md

```diff
@@ -257,7 +262,8 @@ A package can contain any number of **ActionStep** elements, but they must be co
 </Package>
 ```
 
-The above example is an excerpt of a plugin.xml file that defines a plugin that includes a test plan to verify that the steps it provides are working correctly, and a binary executable for generating debug waveform data. It also contains CLI actions that allow further configuration when it is installed, and a CLI action that it needs to run when it is uninstalled. All this is done using the **ActionStep** elements.
+The above example is an excerpt of a plugin.xml file that defines a plugin that includes a test plan to verify that the steps it provides are working correctly, and a binary executable for generating debug waveform data. It also contains CLI actions that allow further configuration when it is installed, and a CLI action that it needs to run before it is uninstalled, and a CLI action that it needs to run when it is uninstalled. All this is done using the **ActionStep** elements.
```

Comment:
Looks like a "typo" that `and a CLI action that it needs to run when it is uninstalled` is now here twice